base/.eslintrc.json

```diff
@@ -1,8 +1,18 @@
 {
+  "plugins": [
+    "prettier"
+  ],
   "extends": [
-    "plugin:prettier/recommended"
+    "eslint-config-prettier",
+    "plugin:prettier/recommended",
+    "eslint:recommended",
+    "prettier",
+    "prettier/standard",
+    "prettier/flowtype"
```

Comment:
Are we using flow anywhere? Might not be necessary to load in the prettier config for flow

Reply:
:truestory: Will remove it!